test/parallel/test-runner-coverage-thresholds.js

```diff
@@ -61,6 +61,8 @@ for (const coverage of coverages) {
     const result = spawnSync(process.execPath, [
       '--test',
       '--experimental-test-coverage',
+      '--no-warnings',
+      '--test-coverage-include=**',
```

Comment:
I have mixed feelings about all these `--test-coverage-include=**` flags.
IMHO, since we're changing the default behaviour, it might make sense to avoid a large number of tests focused on a 'non-default' behaviour.
Although we could address this in a separate PR, I think it would be better, if it makes sense to you, to handle it here to avoid a "test-refactor only" PR later.

WDYT?

cc @nodejs/test_runner @RedYetiDev

Reply:
I also have mixed feelings about it, as ** would also include anything in node_modules. I'm honestly okay with keeping the test files in the coverage report, and having the user exclude them with the exclude flag.

Reply:
In the test files, the `**` is fine.

I've self-addressed my concerns:
My concern was that users that want to include test files would use `--test-coverage-include=**`, which will also include `node_modules/`, which they may not want. I suppose it's not that bad for users who, for some reason, need the test files (and only the test files) to use `--test-coverage-include=** --test-coverage-exclude=node_modules/`.

Reply:
Ahhh I see, that's true, and that's not comfortable neither... I think that the more we are talking, the more this is heading to have an exclusive `--include-test-files` flag that is disabled by default... Makes sense as right now test files is an exclusive option named "testFiles" in the TestCoverage class not linked to the include or exclude in any direct way, but just when applying the filters.

Reply:
Maybe, but that could be done in a followup if users show that they need it. For now, maybe this is okay?

Reply:
> This first goes to the non LTS and when people complain/debate/whatever it goes to the LTS if okay, right?

Yes, that's how it works. I'm happy to see where this goes.